internal/monitors/replicationstatus.go

```diff
@@ -0,0 +1,61 @@
+package monitors
+
+import (
+	"log"
+	"time"
+
+	"cloudant.com/cloudant_exporter/internal/utils"
+	"github.com/IBM/cloudant-go-sdk/cloudantv1"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+type ReplicationStatusMonitor struct {
+	Cldt     *cloudantv1.CloudantV1
+	Interval time.Duration
+	FailBox  *utils.FailBox
```

Comment:
I don't think you need the FailBox or Interval, I think I removed that from the Monitors in a previous push (the monitor looper uses them in main.go)

Reply:
Done